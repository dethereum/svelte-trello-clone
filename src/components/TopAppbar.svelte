<script>
  import { Navbar, NavbarBrand } from "sveltestrap";

  import src from "../../assets/images/cream.png";
</script>

<style>
  /* Make brand responsive */
  /* Extra small devices (portrait phones, less than 576px) */
  @media (max-width: 575px) {
    div > :global(a) {
      font-size: 1rem;
      font-weight: 700;
    }
  }

  /* Small devices (landscape phones, 576px and up) */
  @media (min-width: 576px) {
    div > :global(a) {
      font-size: 1.2rem;
      font-weight: 500;
    }
  }

  /* Extra large devices (large desktops, 1200px and up) */
  @media (min-width: 1200px) {
    div > :global(a) {
      font-size: 1.4rem;
      font-weight: 200;
    }
  }

  img {
    max-width: 2rem;
  }
</style>

<Navbar color="primary" dark class="mb-2 sticky-top">
  <div class="container">
    <NavbarBrand>
      <img class="d-inline-block align-middle" {src} alt="Sally Brand" />
      Sally
    </NavbarBrand>
  </div>
</Navbar>
