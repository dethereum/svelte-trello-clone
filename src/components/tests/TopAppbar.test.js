import { render } from '@testing-library/svelte'

import TopAppbar from '../TopAppbar.svelte'


describe('TopAppbar', () => {
    it('renders correctly', () => {
        const { container } = render(TopAppbar)
        
        expect(container.firstChild).toMatchSnapshot();
    })
})