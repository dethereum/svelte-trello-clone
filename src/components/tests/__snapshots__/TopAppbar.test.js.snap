// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopAppbar renders correctly 1`] = `
<div>
  <nav
    class="mb-2 sticky-top navbar navbar-dark bg-primary"
  >
    <div
      class="container svelte-10nhwqv"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        <img
          alt="Sally Brand"
          class="d-inline-block align-middle svelte-10nhwqv"
        />
        
      Sally
      </a>
    </div>
  </nav>
</div>
`;
